### PR TITLE
Update schedule list in UI to reduce ambiguity

### PR DESCRIPTION
**Pull request by clarud**

### CODE REVIEW

File: src/main/java/seedu/address/commons/util/DateUtil.java [hunk 1]
```diff
@@ -21,6 +21,9 @@ public class DateUtil {
      * @return String containing date in "dd MMMM yyyy" format
      */
     public static String getDisplayableDate(LocalDate date) {
+        if (date == null) {
+            return "null";
+        }
         return date.format(DATE_DISPLAY_FORMATTER);
```
Comment (niclammm) [inline src/main/java/seedu/address/commons/util/DateUtil.java:27]: Good handling of null case

**Review by niclammm — COMMENTED** [review]
LGTM

**Comment by niclammm** [comment]
lgtm

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 2]
```diff
@@ -141,6 +144,7 @@ private void fillPersonsAndAppointments() {
         personListPanel = new PersonListPanel(logic.getFilteredPersonList());
         personListPanelPlaceholder.getChildren().add(personListPanel.getRoot());
 
+        appointmentListLabel.setText(logic.getFilteredAppointmentDate() + " Appointments");
```
[the remark this answers is not in the document]
Comment (choonzies) [reply src/main/java/seedu/address/ui/MainWindow.java:147]: Cleared, it appends "All "

**Pull request merged**